config/application.rb

```diff
@@ -23,24 +24,6 @@ class Application < Rails::Application
     #
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
-    #
-    # config.time_zone = "Central Time (US & Canada)"
-    # config.eager_load_paths << Rails.root.join("extras")
-
-    # Configure the default encoding used in templates for Ruby 1.9.
-    config.encoding = 'utf-8'
-
-    # Configure sensitive parameters which will be filtered from the log file.
-    config.filter_parameters += [:password]
```

Comment:
This is not required, we already have `:passw ` in `config/initializers/filter_parameter_loggin.rb` and Rails uses partial parameter name matching